internal/config/config.go

```diff
@@ -38,6 +38,12 @@ func MustEnableUnitTestingMode() {
 }
 
 func EnableUnitTestingMode() error {
+	return EnableUnitTestingModeFull(true)
+}
+
+// Changes the base directory to be a temp directory to cleanly store testing data in temp dir.
```

Comment:
Bad comment.

This comment is factually accurate, but does not serve the purpose of providing an overview of the function.